lib/Minz/ExtensionManager.php

```diff
@@ -468,9 +396,14 @@ public static function callHookVoid(string $hook_name): void {
 	 * Call a hook which takes no argument and returns nothing.
 	 * Same as callHookVoid but only calls the first extension.
 	 *
-	 * @param string $hook_name is the hook to call.
+	 * @param string|Minz_HookType $hook is the hook to call.
 	 */
-	public static function callHookUnique(string $hook_name): bool {
+	public static function callHookUnique(string|Minz_HookType $hook): bool {
+		if (null === $hook = self::extractHook($hook)) {
+			throw new \RuntimeException("The \"{$hook}\" does not exist!");
```

Comment:
```suggestion
			throw new \RuntimeException("The hook “{$hook}” does not exist!");
```

Minor: typographic quotes

Reply:
done